src/components/Form.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+const Form = ({handleClick}) => {
+    const [chosenIngr, setChosenIngr] = useState('');
+    const handleChange = (event) => {
+        setChosenIngr(event.target.value);
+    }
+
+    return (
+        <section className="container">
+            <form action="">
+                <label>Enter your ingredients separated by a comma! </label>
+                <br></br>
+                <input type="text" className="textInput" placeholder="ex. tomato, brie" onChange={handleChange} value={chosenIngr}></input>
```

Comment:
is this label for the input? don't forget to link them with the htmlFor and name attributes for the label and input elements respectively